pyvo/dal/sia.py

```diff
@@ -878,11 +878,10 @@ def getdataurl(self):
         to retrieve the dataset described by this record.  None is returned
         if no such column exists.
         """
-        dataurl = super().getdataurl()
-        if dataurl is None:
+        if self.acref is not None:
```

Comment:
As painful as it is, I suppose there should be a test for this new behavior.

Reply:
I feared I can't dodge this, and to be fair, I would have pointing out the missing test in my own review 😅

Reply:
interestingly `getdataurl()` is already been tested, I suppose the mock test case is just not prepared carefully enough.